### PR TITLE
GROUNDWORK-2421: Convert all octets metrics to bytes

### PR DESCRIPTION
**Pull request by VladislavSenkevich**
+ add additional in/out per second metrics

### CODE REVIEW

File: connectors/snmp-connector/snmpConnectorModel.go [hunk 1]
```diff
@@ -74,30 +74,36 @@ func (device *DeviceExt) retrieveMonitoredServices(metricDefinitions map[string]
 	timestamp := transit.NewTimestamp()
 	i := 0
 	for _, iFace := range device.Interfaces {
-		var metricsBuilder []connectors.MetricBuilder
+		bytesInPrev, bytesOutPrev, bytesInX64Prev, bytesOutX64Prev := -1, -1, -1, -1
```
Comment (ymkins) [inline connectors/snmp-connector/snmpConnectorModel.go:77]: these vars should be defined as int64

File: connectors/snmp-connector/snmpConnectorModel.go [hunk 2]
```diff
@@ -74,30 +74,36 @@ func (device *DeviceExt) retrieveMonitoredServices(metricDefinitions map[string]
...
+		if val, ok := previousValueCache.Get(fmt.Sprintf("%s:%s:%s", device.Name, iFace.Name, clients.IfInOctets)); ok {
+			bytesInPrev = val.(int)
+		}
+		if val, ok := previousValueCache.Get(fmt.Sprintf("%s:%s:%s", device.Name, iFace.Name, clients.IfOutOctets)); ok {
+			bytesOutPrev = val.(int)
+		}
+		if val, ok := previousValueCache.Get(fmt.Sprintf("%s:%s:%s", device.Name, iFace.Name, clients.IfHCInOctets)); ok {
+			bytesInX64Prev = val.(int)
```
Comment (ymkins) [inline connectors/snmp-connector/snmpConnectorModel.go:85]: should be val.(int64) there

File: connectors/snmp-connector/snmpConnectorModel.go [hunk 3]
```diff
@@ -74,30 +74,36 @@ func (device *DeviceExt) retrieveMonitoredServices(metricDefinitions map[string]
...
+		}
+		if val, ok := previousValueCache.Get(fmt.Sprintf("%s:%s:%s", device.Name, iFace.Name, clients.IfHCOutOctets)); ok {
+			bytesOutX64Prev = val.(int)
```
Comment (ymkins) [inline connectors/snmp-connector/snmpConnectorModel.go:88]: should be val.(int64) there

File: connectors/snmp-connector/snmpConnectorModel.go [hunk 4]
```diff
@@ -38,9 +38,9 @@ type InterfaceExt struct {
 }
 
 type InterfaceMetric struct {
-	Mib      string
-	Value    int
-	UnitType clients.SnmpUnitType
+	Key   string
+	Mib   string
+	Value int
```
Comment (ymkins) [inline connectors/snmp-connector/snmpConnectorModel.go:43]: should be defined as int64

File: connectors/snmp-connector/snmpConnectorModel.go [hunk 5]
```diff
@@ -164,22 +186,36 @@ func calculateValue(metricKind transit.MetricKind, unitType transit.UnitType,
...
-			}
-			return true, false, currentValue
 		}
+		return true, false, currentValue
 	}
 	return false, false, currentValue
 }
+
+func calculateBytesPerSecond(metricName string, metricDefinition transit.MetricDefinition, current, currentX64, previous,
+	previousX64 int, timestamp *transit.Timestamp) connectors.MetricBuilder {
```
Comment (ymkins) [inline connectors/snmp-connector/snmpConnectorModel.go:200]: should be int64 there

**Review by ymkins — CHANGES_REQUESTED** [review]

File: connectors/snmp-connector/snmpConnectorModel.go (hunk 1 above)
Comment (VladislavSenkevich) [reply connectors/snmp-connector/snmpConnectorModel.go]: Done

File: connectors/snmp-connector/snmpConnectorModel.go (hunk 3 above)
Comment (VladislavSenkevich) [reply connectors/snmp-connector/snmpConnectorModel.go]: Done

File: connectors/snmp-connector/snmpConnectorModel.go (hunk 4 above)
Comment (VladislavSenkevich) [reply connectors/snmp-connector/snmpConnectorModel.go]: Done

File: connectors/snmp-connector/snmpConnectorModel.go (hunk 5 above)
Comment (VladislavSenkevich) [reply connectors/snmp-connector/snmpConnectorModel.go]: Done

**Review by ymkins — APPROVED** [review]

**Pull request merged**